simulator/mips/t/unit_test.cpp

```diff
@@ -9,6 +9,15 @@ TEST_CASE( "MIPS32_instr_init: Process_Wrong_Args_Of_Constr")
     CHECK_NOTHROW( MIPS32Instr(0x0).execute());
 }
 
+TEST_CASE( "sllv")
+{
```

Comment:
You test only R-instructions. Could you please add some tests for I, J, RI and other instruction types?